src/utils/getVisiblePeople.ts

```diff
@@ -0,0 +1,56 @@
+import { Person } from '../types';
+import { sortPeople } from './sortPeople';
+
+interface Params {
+  sex: string,
```

Comment:
Consider using more specific types for 'sex' field in 'Params' interface. It seems like it should be a union type (e.g., 'male' | 'female') rather than a generic string.

Reply:
Not fixed. The sex can be 'f' or 'm'. Not just a random string
![image](https://github.com/mate-academy/react_people-table-advanced/assets/99403198/cc7205c7-f092-4393-8f92-b367cde55833)